Source/Particles/Gather/FieldGather.H

```diff
@@ -105,6 +105,10 @@ void doGatherShapeN (const amrex::ParticleReal xp,
     amrex::Real sx_cell[depos_order + 1];
     amrex::Real sx_node_galerkin[depos_order + 1 - galerkin_interpolation];
```

Comment:
Thanks for the fix! I had noticed this warning also. An alternate method is to do

```
amrex::Real sx_node_galerkin[depos_order + 1 - galerkin_interpolation] = {0._rt};
amrex::Real sx_cell_galerkin[depos_order + 1 - galerkin_interpolation] = {0._rt};
```

Reply:
Yep, that's a lesser known feature of array initialization in C++, just for reference:
https://en.cppreference.com/w/c/language/array_initialization

Reply:
Its' much better like this! I didn't know that it was possible to initialize C-style arrays in this way. @dpgrote, if you want to "insert a suggestion" I will accept your commit (otherwise, I can do that).

Reply:
@lucafedeli88 You can make the change.

Reply:
done ! As soon as tests pass I will merge